app/src/main/kotlin/com/wire/android/ui/sharing/ImportMediaAuthenticatedViewModel.kt

```diff
@@ -117,18 +117,16 @@ class ImportMediaAuthenticatedViewModel @Inject constructor(
         }
     }
 
+    @OptIn(ExperimentalCoroutinesApi::class)
     private suspend fun observeConversationWithSearch() = viewModelScope.launch {
```

Comment:
Maybe we should rename now the method to something different?
```suggestion
    private suspend fun fetchConversationWithSearch() = viewModelScope.launch {
```